token-metadata-creator/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
You don't need a LICENSE in every cabal package. Just a single one at the top level.